pkg/pillar/hypervisor/xen.go

```diff
@@ -336,6 +394,14 @@ func (ctx xenContext) CreateDomConfig(domainName string, config types.DomainConf
 		}
 	}
 	if len(pciAssignments) != 0 {
+		c, err := ctx.GetCapabilities()
+		if err != nil {
+			logrus.Errorf("cannot check capabilities: %v", err)
+		} else {
+			if !c.IOVirtualization {
+				return logError("can't set pci assignments: no I/O virtualization support")
+			}
+		}
```

Comment:
Can't we move this to domainmgr and do it only there?

Reply:
Done